### DISCUSSION TITLE
Why "Cannot write file" error in tsconfig.json?

### DISCUSSION THREAD

**Discussion by johanalkstal** [chakra-ui/panda#1193] [Q&A]
I'm using Panda in a SvelteKit application and for some reason I'm getting the following error reported in VSCode from the `tsconfig.json` file.

`Cannot write file '/path-to-project/styled-system/css/conditions.mjs' because it would overwrite input file.`

Everything seems to work and it seems to be an annoyance cause it spams the Problems windows with 27 of these errors.

The `tsconfig.json` file looks like this,

```
{
	"extends": "./.svelte-kit/tsconfig.json",
	"compilerOptions": {
		"allowJs": true,
		"checkJs": true,
		"esModuleInterop": true,
		"forceConsistentCasingInFileNames": true,
		"resolveJsonModule": true,
		"skipLibCheck": true,
		"sourceMap": true,
		"strict": true
	}
	// Path aliases are handled by https://kit.svelte.dev/docs/configuration#alias
	//
	// If you want to overwrite includes/excludes, make sure to copy over the relevant includes/excludes
	// from the referenced tsconfig.json - TypeScript does not merge them in
}
```

Does anyone have a clue what's going on?

**Comment by technorav3nn** [comment]
Did u ever fix it? it happens in nuxt
